manifests/config/puppetserver.pp

```diff
@@ -4,7 +4,11 @@
 ) {
   require ::puppetserver::augeas
 
-  $targetdir = '/etc/puppetserver/conf.d'
+  if versioncmp($::puppetversion, '4.0.0') >= 0 {
```

Comment:
@raphink I guess it's safe to admit that if `puppetversion >= 4.0.0` than we use puppetserver 2.x, so I'm not sure that we need (yet) another fact which will not be available at first run.